infra/nncc/cmake/CfgOptionFlags.cmake

```diff
@@ -26,6 +26,7 @@ option(DOWNLOAD_ABSEIL "Download Abseil-cpp source" ON)
 option(DOWNLOAD_OPENCL_HEADERS "Download OpenCl Header source" ON)
 option(DOWNLOAD_PYBIND11 "Download Pybind11 source" ON)
 option(DOWNLOAD_JSONCPP "Download Jsoncpp source" ON)
+option(DOWNLOAD_LIBNPY "Download Libnpy source" ON)
```

Comment:
let's set this as default OFF for now.
we can change this to ON later when this tool is going to be released.